### ISSUE TITLE
Does not accept list with unmatched parens.

### ISSUE THREAD

**Issue by jasonsikes** [jrincayc/ucblogo-code#52]
In the documentation it states that unmatched parenthesis (among other things) while reading a list will cause a continuation of input, but UCBLogo will accept a closing bracket. The closing bracket prevents UCBLogo from properly accepting a closing parenthesis. For example:
```
? PRINT [ Errors make me sad. :( ]
~ )
unexpected ')'
```

**Comment by Loud-Cat** [comment]
Yeah, luckily it can be escaped with either a backslash or vertical bars

```lg
? PR [Errors make me sad |:(|]
Errors make me sad :(
? PR [Errors make me sad :\(]
Errors make me sad :(
```

However, lists entered through user input require special treatment. `READWORD` and `READRAWLINE` will both read that string with no problems (as a word with spaces), but `READWORD` will continue on the next line if it ends with a tilde.
